spec/javascripts/test_spec.js

```diff
@@ -0,0 +1,15 @@
+/* jshint node: true */
+var basic = require("../../src/object.js");
+describe("Basic suite", function() {
```

Comment:
Hmm, it said "possibly due to async" and that would be this line, if you had a handler like `function(done)` -- but you don't. I'm not sure -- does this run properly on your own machine?